### PR TITLE
Fix typos for ABC command names

### PR DESCRIPTION
**Pull request by mathursanjiv**
Fix typo to use timing instead of delay in option "target"

Signed-off-by: Sanjiv Mathur <mathur.sanjiv@gmail.com>

### CODE REVIEW

File: src/rmp/src/Restructure.cpp [hunk 1]
```diff
@@ -524,20 +522,10 @@ void Restructure::writeOptCommands(std::ofstream& script)
...
-    opt_mode_ = Mode::DELAY_3;
-  else if (!strcmp(mode_name, "delay4"))
-    opt_mode_ = Mode::DELAY_4;
-  else if (!strcmp(mode_name, "area1") || !strcmp(mode_name, "area"))
+  else if (!strcmp(mode_name, "area"))
     opt_mode_ = Mode::AREA_1;
-  else if (!strcmp(mode_name, "area2"))
-    opt_mode_ = Mode::AREA_2;
-  else if (!strcmp(mode_name, "area3"))
-    opt_mode_ = Mode::AREA_3;
```
Comment (maliberty) [inline src/rmp/src/Restructure.cpp:540]: Are the other modes obsolete?

File: src/rmp/src/Restructure.cpp [hunk 2]
```diff
@@ -264,17 +264,15 @@ void Restructure::getEndPoints(sta::PinSet& ends,
...
       sta::PathExpanded expanded(path_end->path(), open_sta_);
       logger_->report("Found path of depth {}", expanded.size() / 2);
       if (expanded.size() / 2 > max_depth) {
         ends.insert(path_end->vertex(sta_state)->pin());
-        end_count++;
       }
-      if (end_count > 5)  // limit blob size for timing
-        break;
+      // Use only one end point to limit blob size for timing
+      break;
```
Comment (maliberty) [inline src/rmp/src/Restructure.cpp:275]: This seems unrelated to typos.  Is this intentional?

File: src/rmp/src/Restructure.cpp (hunk 1 above)
Comment (mathursanjiv) [reply src/rmp/src/Restructure.cpp:540]: Externally we are supporting only 2 modes: timing and area
Within timing and area, the submodes, are not visible externally now. Earlier I was thinking to keeping them external as well.

File: src/rmp/src/Restructure.cpp (hunk 2 above)
Comment (mathursanjiv) [reply src/rmp/src/Restructure.cpp:275]: Yes, there had been feedback to remove 5 as heuristic. With some experimentation done so far, look like keeping it to 1 should be ok. So removing check wrt hard coded value of 5.

**Review by maliberty — APPROVED** [review]
The commit summary is misleading as there are three changes in here.  In the future please be more precise.
